include/picongpu/param/atomicPhysics.param

```diff
@@ -0,0 +1,118 @@
+/* Copyright 2019-2022 Brian Marre
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ * configuration file for atomicPhysics, see speciesDefinition.param for
+ * actual modelling of atomic Physics of ions.
+ */
+
+#pragma once
+
+#include <pmacc/identifier/value_identifier.hpp>
+
+#include <string>
+
+#include <picongpu/particles/atomicPhysics/stateRepresentation/ConfigNumber.hpp>
+#include <picongpu/particles/atomicPhysics/stateRepresentation/LevelVector.hpp>
+
+namespace picongpu
+{
+    namespace atomicPhysics
+    {
+        // datatype/coding used in representation atomic state
+        template<uint8_t T_NumberStates>
+        using LevelVector = particles::atomicPhysics::stateRepresentation::LevelVector<float_X, T_NumberStates>;
+
+        // datatype to be used for configNumber storage
+        // @todo make configureable per species and bind to ionizationEnergy data to avoid
+        // double definitions
+
+        using configNumberDataType = uint64_t;
+        constexpr uint8_t protonNumber = 18u;
+        constexpr uint8_t numLevels = 10u;
+
+        /* pathes of file containing atomic input data,
+         * see documentation for creation[@todo], only one different ion possible for now[@todo]
+         */
+        // state data
+        const std::string stateDataFileName = "/home/marre55/ArgonLevels.txt";
+        // transition data
+        const std::string transitionDataFileName = "/home/marre55/ArgonTransitions.txt";
```

Comment:
Same here, please add a relative path this would give the user to add the required copy command to the config file so that tbg is coping the data.

Reply:
will be done in a future pull reqest